scripts/qa_crowbarsetup.sh

```diff
@@ -1102,17 +1102,6 @@ function onadmin_add_cloud_repo()
 }
 
 
-function do_set_repos_skip_checks()
-{
-    # We don't use the proper pool/updates repos when using a devel build
-    if iscloudver 6plus && [[ $cloudsource =~ develcloud ]]; then
-        export REPOS_SKIP_CHECKS+=" SUSE-OpenStack-Cloud-SLE11-$(getcloudver)-Pool SUSE-OpenStack-Cloud-SLE11-$(getcloudver)-Updates"
-    elif iscloudver 5plus && [[ $cloudsource =~ (develcloud|GM5$|GM6$) ]]; then
-        export REPOS_SKIP_CHECKS+=" SUSE-Cloud-$(getcloudver)-Pool SUSE-Cloud-$(getcloudver)-Updates"
-    fi
```

Comment:
Also, it feels to me like we'll need some magic way to tell crowbar to skip repo checks still...